Tools/cases_generator/instructions.py

```diff
@@ -37,6 +37,7 @@ class ActiveCacheEffect:
     "import_from",
     "import_name",
     "_PyObject_CallNoArgs",  # Proxy for BEFORE_WITH
+    "TIER_ONE_ONLY",
```

Comment:
I wonder if we should just replace this entire list with just `"TIER_ONE_ONLY"`, and annotate the offending instructions.

Reply:
That would make sense. For another PR, though.